webextensions/manifest/browser_specific_settings.json

```diff
@@ -12,16 +12,10 @@
               "version_added": "15",
               "version_removed": "79"
             },
-            "firefox": [
-              {
-                "version_added": "48"
-              },
-              {
-                "version_added": "42",
-                "alternative_name": "applications",
-                "notes": "Mandatory before Firefox 48."
-              }
-            ],
+            "firefox": {
+              "version_added": "42",
+              "notes": "Before Firefox 48, this property was required."
+            },
```

Comment:
Why has this been changed? This breaks our tooling and it took me a while to figure that out since it is not mentioned anywhere in the changelog. This patch is for Safari, it would have been good to not change Firefox data or to move these changes to a separate issue/PR.

Reply:
This PR is actually for Manifest V3 entirely as well, and if you look at the diff, you'll notice that this is simply consolidating the statements together to adhere to the typical structure of BCD (where a behavioral difference that only applies to one browser should be a single note, rather than two separated statements).

We apologize in advance for the changes causing an issue with your tooling; however, please do note that features may be added, removed or modified at any time to match developer interest, BCD data guidelines, or for better accuracy, as explained in the [Semantic versioning policy](https://github.com/mdn/browser-compat-data#Semantic-versioning-policy).  In other words, changes such as this are expected to happen in any release and will not result in any changelog notes.